public/docs/_examples/gettingstarted/ts/src/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Why remove the `tsconfig.json` ?

We still need to transpile the TypeScript and these are the settings we are using for this and ToH.

cc / @alexeagle